src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -450,27 +450,28 @@
                          network-preferences
                          gas-rates
                          from-locked-amount]]
-     {:db            (cond-> db
-                       :always (assoc-in [:wallet :ui :send :amount] amount)
-                       :always (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
-                       :always (assoc-in [:wallet :ui :send :sender-network-values]
-                                sender-network-values)
-                       :always (assoc-in [:wallet :ui :send :receiver-network-values]
-                                receiver-network-values)
-                       :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
-                       :always (update-in [:wallet :ui :send] dissoc :network-links)
-                       token   (assoc-in [:wallet :ui :send :token] token))
-      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
-                       :params     request-params
-                       :on-success (fn [suggested-routes]
-                                     (rf/dispatch [:wallet/suggested-routes-success suggested-routes
-                                                   now]))
-                       :on-error   (fn [error]
-                                     (rf/dispatch [:wallet/suggested-routes-error error])
-                                     (log/error "failed to get suggested routes"
-                                                {:event  :wallet/get-suggested-routes
-                                                 :error  error
-                                                 :params request-params}))}]})))
+     (when (and to-address from-address amount-in token-id)
```

Comment:
What will happen if any of these parameters is missing?

The event handler will just return `nil` and nothing will be performed but we did all of the calculations in the `let` bindings.

Why would we call this event to just perform nothing?
Could you please elaborate on it?
IDK if this is just a safety check, if so, then it's ok, but I think we should also solve the root problem (i.e. the dispatcher passing `nil` values).

Reply:
> Hey @BalogunofAfrica, I have been working on fixing the above notification error, and I realized you are updating the getSuggestedRoutes method in your [PR](https://github.com/status-im/status-mobile/pull/20631/files#diff-089fb8d72eefa455e3dc6437b0bc11c3ca3cc0fb4063e20f129b09929aeae3a9L365-R361), so should I close this one?

@ulisesmac issue is being addressed in Abdul's PR

To answer your question, this event was unexpectedly being called after completing a send transaction (with nil values) resulting in a notification error. The root fix is not to call it after a send transaction (fixed in routeV2), and the `nil` check is just for safety to not return a notification error